### PR TITLE
LPP-55697 Default locale creating a web content

### PR DESCRIPTION
**Pull request by DiegoHu97**
LPP-55697 Default locale creating a web content

# What is this trying to solve?

Fix the issue reported in the LPP.

[Link to ticket](https://liferay.atlassian.net/browse/LPD-37464)

# How am I fixing it?

- Remove the part that inherits the default locale from the structue and always inherit from the site, as the structure also inherits it from the site

# How can I test it?

Steps: 

- Follow the steps in the ticket

### CODE REVIEW

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/journal/journal-web/src/test/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContextTest.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ public void testFolderNameValueIsCached() {
 
 	@Test
 	public void testGetDefaultArticleLanguageId() throws PortalException {
-		_testGetDefaultArticleLanguageIdFromArticle();
+		_testGetDefaultArticleLanguageIdWithEmptyeLocale();
```
Comment (jkappler) [inline modules/apps/journal/journal-web/src/test/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContextTest.java:106]: @DiegoHu97, SF :P

**Review by jkappler — CHANGES_REQUESTED** [review]

**Comment by DiegoHu97** [comment]
ci:test:content-management

**Comment by DiegoHu97** [comment]
### ci:test:content-management

**Comment by DiegoHu97** [comment]
ci:test:content-management

**Comment by DiegoHu97** [comment]
ci:report:23052793

**Comment by DiegoHu97** [comment]
ci:test:content-management

**Comment by DiegoHu97** [comment]
ci:report:24302022

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:content-management

**Comment by DiegoHu97** [comment]
ci:report:27805939

**Comment by DiegoHu97** [comment]
ci:report:27805939

**Comment by DiegoHu97** [comment]
@jkappler I reviewed the failures, they are not related, could you review the pr? :heart:

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Comment by DiegoHu97** [comment]
ci:stop

**Comment by DiegoHu97** [comment]
Closing this as it's not a bug

**Pull request closed without merging**